docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+- `Admin.TLS.Enabled:` - This value must be set to `true`.
+- `Admin.TLS.PrivateKey:` - The path to and file name of the orderer private key issued by the TLS CA.
+- `Admin.TLS.Certificate:` - The path to and file name of the orderer signed certificate issued by the TLS CA.
+- `Admin.TLS.RootCAs:`  - The path to and file name of the orderer TLS CA Root certificate.
+- `Admin.TLS.ClientAuthRequired:` This value must be set to `true`. Note that while mutual TLS is required for all operations on the orderer Admin endpoint, the entire network is not required to use Mutual TLS.
+- `Admin.TLS.ClientRootCAs:` - The path to and file name of the admin client TLS CA Root certificate. In the folder structure above, this is `admin-client/client-tls-ca-cert.pem`.
+- `ChannelParticipation.Enabled:` - Set this value to `true` to enable this feature on the orderer.
+
+**Start each orderer**  
+
+1. If you have not already, set the path to the location of the Fabric binaries on your system:
+    ```
+    export PATH=<path to download location>/bin:$PATH
+    ```
+2. In your terminal window set the `FABRIC_CFG_PATH` to point to the location of the `orderer.yaml` file, relative to where you are running the Fabric commands from. For example, if you download the binaries, and run the commands from the `/bin` directory and the `orderer.yaml` is under `/config`, the path would be:
+    ```
+    export FABRIC_CFG_PATH=../config
+    ```
+3. You can now start the ordering service by running the following command on each ordering node:
+    ```
+    orderer start
+    ```
+
+When the ordering service starts successfully, you should see something similar to the following output:
+```
+INFO 01d Registrar initializing without a system channel, number of application channels: 0, with 0 consensus.Chain(s) and 0 follower.Chain(s)
+INFO 01f Starting orderer:
+```
+This action starts the ordering nodes without any channels. Repeat these steps for each orderer. At this point, the three nodes are not communicating with each other until we create a channel in the subsequent steps.
+
+### Define your peer organizations
+
+Because the channel you are creating is meant to be used by two or more peer organizations to transact privately on the network, you need to have at least one peer organization defined to act as the channel administrator who can add other organizations. Technically, the peer nodes themselves do not yet have to be deployed, but you do need to create one or more peer organization MSP definitions and at least one peer organization needs to be provided in the `configtx.yaml` in the next step. Before proceeding to the next section, follow the steps in the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#create-the-org-msp-needed-to-add-an-org-to-a-channel) documentation to build your peer organization MSP definition. If the peers have been deployed, you should also include their address in the `AnchorPeers:` section.
```

Comment:
"If the peers have been deployed, you should also include their address in the `AnchorPeers:` section."

Is this still true? I thought since configtxgen flag --outputAnchorPeersUpdate is deprecated, that we wouldn't recommend this anymore. May need @jyellick opinion.

Reply:
`--outputAnchorPeersUpdate` is deprecated because it's not possible to reliably compute a safe and correct update without knowing the original configuration.

There is nothing strictly wrong with specifying anchor peers in a genesis block, because there is no delta, and no lost safety/correctness.

That being said, it's always struck me as odd/wrong from a workflow perspective to have anchor peers in the genesis block.  This is because inherently, the peer cannot be joined to the channel before the block is created, and, the anchor peer is incorrect until the peer has been joined.  That being said, for the purposes of configuring a working network (with gossip) in as few steps as possible, I think it's probably fine.

Reply:
OK. I'll leave it in for now.